### PR TITLE
Lists,text utilities

### PR DESCRIPTION
**Pull request by shrushti2000**
#### Lists
1. simple
2. stacked

#### Text Utilities
Added different styles for texts.

[Click here to watch live preview](https://focused-saha-a81dcb.netlify.app/adornui-documentation/list)

### CODE REVIEW

File: AdornUI-Stylesheets/TextUtilities.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  font-size: 2rem;
+}
+
+h5 {
+  font-size: 1.75rem;
+}
+
+h6 {
+  font-size: 1.2rem;
+}
```
Comment (viru2001) [inline AdornUI-Stylesheets/TextUtilities.css]: you can make classes for headings

File: AdornUI-Stylesheets/TextUtilities.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+.text {
+  font-size: 2rem;
+  color: var(--black);
+  font-weight: 600;
```
Comment (viru2001) [inline AdornUI-Stylesheets/TextUtilities.css:51]: you can declare variables for font-sizes in your main CSS file if you want

File: AdornUI-documentation/Text Utilities.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        <aside class="sidebar">
+            <h2 class="sidebar-heading">Components</h2><span><i class="fa fa-angle-down"></i></span>
+            <div class="sidebar-items">
+                <a class="sidebar-item-link" href="alert.html">Alerts</a>
+                <a class="sidebar-item-link" href="badges.html">Badges</a>
+                <a class="sidebar-item-link" href="avatar.html">Avatar</a>
+                <a class="sidebar-item-link" href="card.html">Cards</a>
+                <a class="sidebar-item-link" href="button.html">Buttons</a>
+                <a class="sidebar-item-link" href="image.html">Image</a>
+                <a class="sidebar-item-link" href="input.html">Input</a>
```
Comment (viru2001) [inline AdornUI-documentation/Text Utilities.html:32]: you can use relative paths here . 

Refer this for more information 
https://stackoverflow.com/questions/24028561/relative-path-in-html

**Review by viru2001 — COMMENTED** [review]
image

The components list is not visible here...check this once

Also , I suggested some changes which you can do

File: AdornUI-Stylesheets/TextUtilities.css [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+.text-mark {
```
Comment (xii-vi) [inline AdornUI-Stylesheets/TextUtilities.css:56]: What is text-mark ?

File: AdornUI-Stylesheets/list.css [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  font-size: 3rem;
+  border-bottom: 1px solid black;
+  text-decoration: none;
+  list-style: none;
+  font-weight: 900;
+}
+.list-item {
+  font-size: 2rem;
+  text-decoration: none;
+  list-style: none;
```
Comment (xii-vi) [inline AdornUI-Stylesheets/list.css:16]: You are repeating the same property. So try to make a generic class out of it.

**Review by xii-vi — APPROVED** [review]
Added few suggestions.
Rest is okay to merge.

**Pull request merged**